core/src/test/scala/integration/kafka/api/ConsumerBounceTest.scala

```diff
@@ -319,17 +327,20 @@ class ConsumerBounceTest extends AbstractConsumerTest with Logging {
     * the group should be forced to rebalance when it becomes hosted on a Coordinator with the new config.
     * Then, 1 consumer should be left out of the group.
     */
-  @Test
+  @ParameterizedTest(name = TestInfoUtils.TestWithParameterizedQuorumAndGroupProtocolNames)
+  @MethodSource(Array("getTestQuorumAndGroupProtocolParametersAll"))
   @Disabled // TODO: To be re-enabled once we can make it less flaky (KAFKA-13421)
```

Comment:
KAFKA-13421 is still open, so I assume that's why we can't get rid of the `@Disabled` annotation, right?